crates/database/src/jit_orders.rs

```diff
@@ -32,7 +32,7 @@ NULL AS ethflow_data,
 NULL AS onchain_user,
 NULL AS onchain_placement_error,
 COALESCE((SELECT SUM(executed_fee) FROM order_execution oe WHERE oe.order_uid = o.uid), 0) as executed_fee,
-COALESCE((SELECT executed_fee_token FROM order_execution oe WHERE oe.order_uid = o.uid), o.sell_token) as executed_fee_token, -- TODO surplus token
+COALESCE((SELECT executed_fee_token FROM order_execution oe WHERE oe.order_uid = o.uid LIMIT 1), o.sell_token) as executed_fee_token, -- TODO surplus token
```

Comment:
I have the full query compiled. Tested to be 100% sure. Works fine. But should we use `SELECT DISTINCT oe.executed_fee_token` here instead? This would fail if there were multiple different executed fee tokens for the same order, which would signal that something was definitely wrong. While limiting ignores this.

Reply:
I think it's too early to use DISTINCT here for the purpose you explained.

Right now we save all fees in sell token so for a single order, these are guaranteed to be same.
Later, when we switch to saving fee in surplus token, we will potentially have orders with different tokens - but this will be a temporary bug that I explained here https://github.com/cowprotocol/services/pull/2966#discussion_r1758474207
Once I do manual db migration to convert all historical fees to be expressed over surplus token, then again all fees will be the same for a single order, and we can use DISTINCT again.

Reply:
But yes, you are right, we CAN use "SELECT DISTINCT" instead at this point.

But I'd rather not fail the whole query for this